guard/src/commands/validate.rs

```diff
@@ -112,13 +129,16 @@ impl Validate {
     }
 }
 
+const OUTPUT_FORMAT_VALUE_TYPE: [&str; 3] = ["json", "yaml", "single-line-summary"];
```

Comment:
Should we move these to `guard/src/commands/mod.rs`?

Reply:
no i dont think so, since it seems to only be relevant to the validate command